lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -29,13 +29,42 @@ __PACKAGE__->add_columns(
         data_type   => 'text',
         is_nullable => 0,
     },
+    parent_id => {
+        data_type      => 'integer',
+        is_foreign_key => 1,
+        is_nullable    => 1,
+    },
     size_limit_gb => {
         data_type     => 'integer',
         default_value => 100,
     },
     keep_logs_in_days => {
         data_type     => 'integer',
         default_value => 30,
+    },
+    keep_important_logs_in_days => {
+        data_type     => 'integer',
+        default_value => 120,
+    },
+    keep_results_in_days => {
+        data_type     => 'integer',
+        default_value => 200,
+    },
+    keep_important_results_in_days => {
+        data_type     => 'integer',
+        default_value => 365,
```

Comment:
Ok, so maybe 2000 (approx 5.5 years)?
If really only the admin should decide, we could also use the max INT value for infinity.